### ISSUE TITLE
Engine is too restrictive

### ISSUE THREAD

**Issue by JohnAlbin** [ronilaukkarinen/gulp-stylelint#1]
I'm on Node.js v14 and receive this warning during install:

```
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: '@ronilaukkarinen/gulp-stylelint@14.0.5',
npm WARN EBADENGINE   required: { node: '>=15.14.0' },
npm WARN EBADENGINE   current: { node: 'v14.18.1', npm: '8.1.0' }
npm WARN EBADENGINE }
```

The warning is due to this project's `"engines": { "node": ">=15.14.0" },` restriction.

The non-forked version of gulp-stylelint uses:

```
"engines": {
    "node": ">=10.12.0"
  },
```

According to [Node's release page](https://nodejs.org/en/about/releases/), Node 12, 14, 16, and 17 are the supported releases. Only the latest odd-numbered release is supported, so v15 isn't on its officially supported list.

What's the reason for not supporting the Node.js v12 and v14 releases?

**Comment by rollecode** [comment]
As discussed [here](https://github.com/ronilaukkarinen/gulp-stylelint/pull/2) it's a version we ended up for a reason I don't remember any more, somehow left undocumented.

I'll look into this soon. Sorry for the delay, the notification slipped from my radar. Been kinda busy.

**Comment by rollecode** [comment]
@JohnAlbin Thanks for the issue. This is now fixed. Sorry again for it taking it so long.
